src/components/Inventory/index.tsx

```diff
@@ -154,6 +154,8 @@ export default function Inventory(props: InventoryProps): JSX.Element {
               temporalSearchValue={temporalSearchValue}
               setTemporalSearchValue={setTemporalSearchValue}
             />
+          ) : searchResults === null ? (
+            <CircularProgress />
```

Comment:
the initial search on this page was flashing empty state - this is to show initial search is in progress

Reply:
ok, hope this will be easy to rebase